mlir/test/mlir-tblgen/openmp-ops-verify.td

```diff
@@ -0,0 +1,143 @@
+// Tablegen tests for the verification of clause-based OpenMP dialect operation
+// definitions.
+
+// Run tablegen to generate OmpCommon.td in temp directory first.
+// RUN: mkdir -p %t/mlir/Dialect/OpenMP
+// RUN: mlir-tblgen --gen-directive-decl --directives-dialect=OpenMP \
+// RUN:   %S/../../../llvm/include/llvm/Frontend/OpenMP/OMP.td \
+// RUN:   -I %S/../../../llvm/include > %t/mlir/Dialect/OpenMP/OmpCommon.td
```

Comment:
Non-blocking: any chance of threading this through LIT so we don't use relative paths going up too much?

Reply:
Do you mean defining some other substitution and use it in place of "%S"?

Reply:
I figured out a better way, so this should be done now.